### PR TITLE
create class EntityArithmeticInlineTest

### PR DESCRIPTION
**Pull request by PRoman-86**
https://trello.com/c/bkXsYCzy

### CODE REVIEW

**Review by LiubovSuvorova — APPROVED** [review]

File: src/test/java/EntityArithmeticInlineTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        setValue1(randomator());
+        setValue2(randomator());
+        getDriver().findElement(By.xpath("//*[@id=\"f1\"]")).sendKeys(Integer.toString(getValue1()));
+        getDriver().findElement(By.xpath("//*[@id=\"f2\"]")).sendKeys(Integer.toString(getValue2()));
+
+        getDriver().findElement(By.id("pa-entity-form-save-btn")).click();
+        getDriver().findElement(By.xpath("//*[@id=\"pa-all-entities-table\"]/tbody/tr/td[8]/div/button")).click();
+        getDriver().findElement(By.xpath("//*[@id=\"pa-all-entities-table\"]/tbody/tr/td[8]/div/ul/li[1]/a")).click();
+
+        //get values, set variables
```
Comment (DudeAlex) [inline src/test/java/EntityArithmeticInlineTest.java:32]: Store your locators in separate private static final variables

**Review by SergeiDemyanenko — APPROVED** [review]

File: src/test/java/EntityArithmeticInlineTestByRoman.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    @Test //test by Poptsov Roman
+    public void testCreateAndSaveNewRecord() {
+
+        ProjectUtils.start(getDriver());
+        TestUtils.scrollClick(getDriver(), getDriver().findElement(By.xpath("//*[@id=\"menu-list-parent\"]/ul/li[16]/a/p")));
+        TestUtils.jsClick(getDriver(), getDriver().findElement(By.xpath("/html/body/div/div[2]/div[1]/div/div/div/div/div/div/div/div[1]/a/div/i")));
+        setValue1(randomator());
+        setValue2(randomator());
+        getDriver().findElement(By.xpath("//*[@id=\"f1\"]")).sendKeys(Integer.toString(getValue1()));
```
Comment (GalinaRuban) [inline src/test/java/EntityArithmeticInlineTestByRoman.java:25]: not the best idea to use asteriks * in your locators when you're looking for just one element

File: src/test/java/EntityArithmeticInlineTestByRoman.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        getDriver().findElement(By.xpath("//*[@id=\"f2\"]")).sendKeys(Integer.toString(getValue2()));
+
+        getDriver().findElement(By.id("pa-entity-form-save-btn")).click();
+        getDriver().findElement(By.xpath("//*[@id=\"pa-all-entities-table\"]/tbody/tr/td[8]/div/button")).click();
+        getDriver().findElement(By.xpath("//*[@id=\"pa-all-entities-table\"]/tbody/tr/td[8]/div/ul/li[1]/a")).click();
+
+        //get values, set variables
+        setCurrentSUM(Integer.parseInt(getDriver().findElement(By.xpath("/html/body/div/div[2]/div[1]/div/div/div/div/div[2]/div/div/div/div[3]/div/span")).getText()));
```
Comment (GalinaRuban) [inline src/test/java/EntityArithmeticInlineTestByRoman.java:33]: such locators are not stable, should be modified

File: src/test/java/EntityArithmeticInlineTestByRoman.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        setCurrentSUB(Integer.parseInt(getDriver().findElement(By.xpath("/html/body/div/div[2]/div[1]/div/div/div/div/div[2]/div/div/div/div[4]/div/span")).getText()));
```
Comment (GalinaRuban) [inline src/test/java/EntityArithmeticInlineTestByRoman.java:34]: the same here

File: src/test/java/EntityArithmeticInlineTestByRoman.java [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        setCurrentMUL(Integer.parseInt(getDriver().findElement(By.xpath("/html/body/div/div[2]/div[1]/div/div/div/div/div[2]/div/div/div/div[5]/div/span")).getText()));
```
Comment (GalinaRuban) [inline src/test/java/EntityArithmeticInlineTestByRoman.java:35]: the same here

File: src/test/java/EntityArithmeticInlineTestByRoman.java [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+        setCurrentDIV(Double.parseDouble(getDriver().findElement(By.xpath("/html/body/div/div[2]/div[1]/div/div/div/div/div[2]/div/div/div/div[6]/div/span")).getText()));
```
Comment (GalinaRuban) [inline src/test/java/EntityArithmeticInlineTestByRoman.java:36]: the same here

File: src/test/java/EntityArithmeticInlineTestByRoman.java (hunk 2 above)
Comment (PRoman-86) [reply src/test/java/EntityArithmeticInlineTestByRoman.java:25]: Ок, will correct

**Pull request closed without merging**